addons/traefik/1.7.x/traefik-1.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
     appversion.kubeaddons.mesosphere.io/traefik: "1.7.2"
     endpoint.kubeaddons.mesosphere.io/traefik: "/ops/portal/traefik"
     docs.kubeaddons.mesosphere.io/traefik: "https://docs.traefik.io/v1.7"
-    values.chart.helm.kubeaddons.mesosphere.io/traefik: "https://raw.githubusercontent.com/helm/charts/c7e31361a28a10e30b4143b73439080d5e9f7d8b/stable/traefik/values.yaml"
+    values.chart.helm.kubeaddons.mesosphere.io/traefik: "https://raw.githubusercontent.com/mesosphere/charts/0446b8c4538dba2bb8919a6203b2b8b6c6450365/staging/traefik/values.yaml"
```

Comment:
Wait, were we deploying the upstream chart and now deploy our fork of it?

Reply:
We were not deploying the upstream chart, see below for the `chartReference`.

Reply:
Yes because `useCertManager` doesn't exist in the upstream chart.

Reply:
Oh, that's just the annotation. @shaneutt, do you happen to know what this annotation is used for?